libpod/oci.go

```diff
@@ -106,8 +106,17 @@ func newOCIRuntime(name string, paths []string, conmonPath string, runtimeCfg *R
 		}
 		foundPath = true
 		runtime.path = path
+		logrus.Debugf("using runtime %q", path)
 		break
 	}
+
+	// Search the $PATH as last fallback
+	if foundRuntime, err := exec.LookPath(name); !foundPath && err == nil {
```

Comment:
Might be better to wrap this entire block in `!foundPath` to avoid the lookup in cases where we have a valid path

Reply:
Yes, I adapted as suggested.